docs/acl.md

```diff
@@ -0,0 +1,72 @@
+# Indexing the ACL Anthology with Anserini
+
+Anserini provides code for indexing the ACL anthology with Solr in order to host an ACL anthology [Blacklight](https://github.com/projectblacklight/blacklight) instance.
+
+## Deploying Solr
+
+Set up Solr locally by following the instructions for setting up a single-node SolrCloud instance in [solrini.md](solrini.md).
+
+## Generating ACL Anthology Data
+
+First, clone the ACL anthology repository containing the raw XML data:
+
+  ```git clone git@github.com:acl-org/acl-anthology.git```
+ 
+Next, navigate to the `acl-anthology` folder and install dependencies (with Python 3.7+):
+
+```pip install -r requirements.txt```
+
+Generate cleaned YAML data (again with Python 3.7+):
+
+  ```python bin/create_hugo_yaml.py```
+
+Generated ACL files can now be found in `acl-anthology/build/data/`
+
+## Indexing Data
+
+Once we have a local instance of SolrCloud up and running, we can index using Solr through Solrini.
+
+First, let's create the index collection in Solr using the default config.
+
+```solrini/bin/solr create -n anserini -c acl```
+
+Modify the `acl` Solr schema to allow multiValued fields for facets and set explicit field types:
+
+```sh
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "add-field":{
```

Comment:
Make sense to put in a file in `src/resources`?

Reply:
I had that in my first commit, but then I realized it's just a single command they could copy paste so I pulled it into the README. I'll move it back since it's probably better to make a new file in case things need to get added in the future?

Reply:
I agree.